exercises/list-ops/src/test/java/ListOpsTest.java

```diff
@@ -109,187 +107,168 @@ public void shouldFilterNonEmptyList() {
+                Arrays.asList(0, 1, 2, 3, 4)
         );
         final List<Integer> list2 = Collections.unmodifiableList(
-                IntStream.range(1, 6).boxed().collect(Collectors.toList())
+                Arrays.asList(1, 2, 3, 4, 5, 6)
         );
         final List<Integer> expected
                 = Stream.concat(list1.stream(), list2.stream())
                         .collect(Collectors.toList());
         final List<Integer> actual = ListOps.concat(list1, list2);
-
+        
         assertNotNull(actual);
         assertFalse(actual.isEmpty());
         assertEquals(expected, actual);
     }
-
+    
     @Test
     @Ignore
     public void shouldConcatenateSeveralLists() {
         final List<Integer> list1 = Collections.unmodifiableList(
-                IntStream.range(0, 4).boxed().collect(Collectors.toList())
+                Arrays.asList(0, 1, 2, 3)
         );
         final List<Integer> list2 = Collections.unmodifiableList(
-                IntStream.range(4, 8).boxed().collect(Collectors.toList())
+                Arrays.asList(4, 5, 6, 7)
         );
         final List<Integer> list3 = Collections.unmodifiableList(
-                IntStream.range(8, 12).boxed().collect(Collectors.toList())
+                Arrays.asList(8, 9, 10, 11)
         );
         final List<Integer> list4 = Collections.unmodifiableList(
-                IntStream.range(12, 16).boxed().collect(Collectors.toList())
+                Arrays.asList(12, 13, 14, 15)
         );
         final List<Integer> expected
-                = new ArrayList<>(list1);
-        expected.addAll(list2);
-        expected.addAll(list3);
-        expected.addAll(list4);
-
+                = Arrays.asList(0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13,
+                        14, 15);
+        
         final List<Integer> actual
                 = ListOps.concat(list1, list2, EMPTY_LIST, list3, list4);
-
+        
         assertNotNull(actual);
         assertFalse(actual.isEmpty());
         assertEquals(expected, actual);
     }
-
+    
     @Test
     @Ignore
     public void shouldReturnIdentityWhenAnEmptyListIsReduced() {
         final int expected = 0;
         final int actual
                 = ListOps.reduce(EMPTY_LIST, 0, (x, y) -> x + y, Integer::sum);
-
+        
         assertEquals(expected, actual);
     }
-
+    
     @Test
     @Ignore
     public void shouldCalculateTheSumOfANonEmptyIntegerList() {
         final List<Integer> list = Collections.unmodifiableList(
-                IntStream.range(0, 5)
-                        .boxed()
-                        .collect(Collectors.toList())
+                Arrays.asList(0, 1, 2, 3, 4)
         );
         final int actual = ListOps.reduce(list, 0,
                 (x, y) -> x + y,
                 Integer::sum);
-
+        
         assertEquals(10, actual);
     }
-
+    
     @Test
     @Ignore
     public void shouldReduceWithAnticommutativeAccumulator() {
```

Comment:
🤔  coming back to this topic with a "fresh" mind... mentioning "parallelization" without a requirement to support parallelization in the exercise is a likely source of confusion.  That's a problem.

I see two paths:

1. add the requirement to be able to parallelize stream operations.
2. remove the notion of parallelization from *this* exercise (and attendant APIs that are required by it).

We've packed *so much* into this exercise, as it is, I'm personally leaning toward option 2.

@javaeeeee, do you see another option?  What can we do to avoid this confusion?